kafka-to-kv/kafka-to-kv.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import json
+import base64
+
+import requests
+
+NGINX_HOST = os.environ['NGINX_HOST']
+NGINX_PORT = os.environ['NGINX_PORT']
+TABLE_NAME = os.environ['TABLE_NAME']
+CONTAINER_NAME = os.environ['CONTAINER_NAME']
+EVENT_KEY = os.environ['EVENT_KEY']
+USERNAME = os.environ['USERNAME']
+PASSWORD = os.environ['PASSWORD']
+
+def get_request_url():
+
+    return 'http://{0}:{1}/{2}/{3}/'.format(NGINX_HOST, NGINX_PORT, CONTAINER_NAME, TABLE_NAME)
+
+
+def create_encoded_auth():
```

Comment:
Let's simplify and use an off the shelf package for this. requests can also do with `HTTPBasicAuth`